metacat-functional-tests/src/functionalTest/groovy/com/netflix/metacat/MetacatSmokeSpec.groovy

```diff
@@ -478,6 +481,7 @@ class MetacatSmokeSpec extends Specification {
             if (repeat) {
                 assert api.getTable(catalogName, databaseName, tableName, false, true, false).getDefinitionMetadata().get('savePartitions') != null
                 assert partitions.get(0).dataUri == uri + '0' + uriResult
+                assert partitions.get(0).getDataMetadata().get('updateMetadata') != null
```

Comment:
How is this testing this feature? You should get the partition creation date before saving and validate that it does not change after saving with the saveMetadataOnly flag set to true.